maven-plugins/bnd-resolver-maven-plugin/src/it/resolve-with-runstartlevel-and-runbundles-decorator/invoker.properties

```diff
@@ -0,0 +1,7 @@
+invoker.goals=--no-transfer-progress package
```

Comment:
Hmm. I am not too happy this testing takes place in the maven sub project. The functional change is in bnd so it should be tested in the normal bnd testing. The biz.aQute.resolve.RunResolutionTest in biz.aQute.resolve contains examples.

I know it is more comfortable to stay in your comfort zone but bnd is a shared project. If you change a function in bnd the test must take place as close as possible to the change.

Reply:
That makes sense, I will look into adding test case(s) in RunResolutionTest. Do you want to retain these maven sub project test as well for this particular case, or just the tests within biz.aQute.resolve?

Reply:
I think it is better to test it once in bnd. I consider ANY error that would be detected in a 'driver' (maven, gradle, eclipse, intellij) an error of the highest severity. We rarely have them fortunately. And in this case, the functionality is clearly in bnd, not the driver.

Reply:
Ok, I have removed the integration tests from the maven sub module.